scalding-serialization/src/main/scala/com/twitter/scalding/serialization/OrderedSerialization.scala

```diff
@@ -101,6 +101,30 @@ object OrderedSerialization {
     case NonFatal(e) => CompareFailure(e)
   }
 
+  def viaTransform[T, U](
```

Comment:
Think it probably makes sense to just make this the base/normal variant. We only hit the Try when we are doing deserialization, wherein we have a try anyway. Not sure there is likely much overhead to another allocation for this path?